src/components/navbar/Navbar.jsx

```diff
@@ -22,6 +25,11 @@ const Navbar = () => {
           />
         </IconWrapper>
         <div className="flex">
+          <div className="profile">
+            {
+              authState.isLogedIn ? <button className="btn-grad-red radius-md py-1 px-2" onClick={()=>logOut()}>Logout</button> : <Link to="/login" className="btn-grad-red radius-md py-1 px-2 flex center">Login</Link>
```

Comment:
an additional thing you can do is you can get firstName from the user which we get when we get the token and you can display "Hello , name of the user" on right of the logout button

Reply:
I will  do it for now I'm testing logout functionality